setup.py

```diff
@@ -47,6 +47,7 @@
     description="Low-level communication layer for PRAW 4+.",
     extras_require=extras,
     install_requires=["requests >=2.6.0, <3.0"],
+    python_requires=">=3.5,<4",
```

Comment:
I don't see why we need to have a `<4`, because I highly doubt there will be a Python4 anytime soon, but other than that, I'm surprised this already wasn't part of the package.
```suggestion
    python_requires=">=3.5",
```